### PR TITLE
New landing page

### LINKED ISSUE #71: Improve prism language file
We need to improve the prism language file based from our vscode highlighting syntax.

 - [vscode/syntaxes/story.tmLanguage.json](https://github.com/storyscript/vscode/blob/master/syntaxes/story.tmLanguage.json)

Contributions welcomed

### PR DESCRIPTION
**Pull request by Arinono**
- [x] Init new Vue project with Tailwind and theme setup (colors, fonts, ...)
- [x] Preparing the layout (based routing)
- [x] Percy setup
- [x] Layout:
  - [x] Footer
  - [x] Navbar
    - [x] close menu on click
-  [x] Home:
  - [x] Header
  - [x] Code
  - [x] Data
  - [x] Community
  - [x] Stories
  - [x] A Better Way
  - [x] Ambitious ~(Missing background)~
  - [x] Newsletter
---

New https://storyscript.com website
https://www.figma.com/file/N8oB2hmgThgX4gVQ1ZdKVb/Storyscript-Marketing-Landing-Page

https://www.pivotaltracker.com/story/show/170220136

---

Closes all dependency update because they're manually updated.

Closes #152 #154 #155 #156 #158 #159 #160 #162 #163 #164 #165 #167 #168 #170 

Also closes theses issues because they're now not relevant anymore.

Closes #71 #143 #144 #145 #146 #150

### CODE REVIEW

File: src/assets/styles/custom/_fonts.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+@font-face {
+  font-family: Graphik;
+  src: url('~@/assets/fonts/Graphik/Graphik-Regular.ttf');
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Graphik;
+  src: url('~@/assets/fonts/Graphik/Graphik-Medium.ttf');
```
Comment (JeanBarriere) [inline src/assets/styles/custom/_fonts.scss]: maybe we don't need all of these fonts in the repo for such a small load hahaha

File: src/views/Home/Ambitious.vue [hunk 2]
```diff
@@ -0,0 +1,738 @@
+<template>
+  <div class="flex items-start">
+    <svg
```
Comment (JeanBarriere) [inline src/views/Home/Ambitious.vue]: can you move this svg into another sub component please ? Just so this component is still readable

File: src/views/Layout/Link.vue [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        v-text="link.name"
+      />
+    </a>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Prop } from 'vue-property-decorator'
+import SText from '@/components/Text.vue'
+import { INav } from '../../models/Footer'
```
Comment (JeanBarriere) [inline src/views/Layout/Link.vue]: @/models ?

File: vue.config.js [hunk 4]
```diff
@@ -1,31 +1,5 @@
-const path = require('path')
-const SitemapPlugin = require('sitemap-webpack-plugin').default
```
Comment (JeanBarriere) [inline vue.config.js]: The sitemap disappeared. I don't know if it's wanted

File: src/views/Layout/Link.vue (hunk 3 above)
Comment (Arinono) [reply src/views/Layout/Link.vue]: Auto-import missed this one

**Pull request merged**